hw/dv/tools/dvsim/testplans/sec_cm_count_testplan.hjson

```diff
@@ -0,0 +1,29 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: prim_count_check
+      desc: ''' Verify that violating prim_count counter properties generate a fatal alert.
+
+            Stimulus:
+            - At the falling edge (non-active edge), force the counter to a different value than
+              expected.
+            - Randomly flip the value back to any other value to ensure the error is latched and
```

Comment:
Not sure about this step - it it different from the first?

Reply:
This is to flip the counter to normal value. Reworded. LMK If it's still not clear.
Step 2 of the stimulus at [slides](https://docs.google.com/presentation/d/1hcS2hipBXW4TmIna8xfn83mfk2xIMekxE2xERhD_i38/edit#slide=id.gf56c479c22_0_5)